meta-leda-components/recipes-sdv/eclipse-leda/leda-contrib-self-update-agent_git.bb

```diff
@@ -0,0 +1,65 @@
+# /********************************************************************************
+# * Copyright (c) 2022 Contributors to the Eclipse Foundation
+# *
+# * See the NOTICE file(s) distributed with this work for additional
+# * information regarding copyright ownership.
+# *
+# * This program and the accompanying materials are made available under the
+# * terms of the Apache License 2.0 which is available at
+# * https://www.apache.org/licenses/LICENSE-2.0
+# *
+# * SPDX-License-Identifier: Apache-2.0
+# ********************************************************************************/
+
+SUMMARY = "Self Update Agent offers remote OS updates for edge devices using RAUC"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=a832eda17114b48ae16cda6a500941c2"
+
+SRC_URI = "gitsm://github.com/SoftwareDefinedVehicle/leda-contrib-self-update-agent-fork.git;protocol=https;branch=main"
+SRCREV = "db7ad3935bf2f1fae8207044e15cc78fc6d2824e"
+
+SRC_URI += " \
+    file://self-update-agent/self-update-agent.service \
+    file://self-update-agent/0001-CMakeLists-root.patch \
+    file://self-update-agent/0002-CMakeLists-3rdparty.patch \
+    file://self-update-agent/0003-CMakeLists-src.patch \
+"
+
+S = "${WORKDIR}/git"
+
+inherit cmake pkgconfig systemd
+
+EXTRA_OECMAKE += "-DSUA_COMMIT_HASH=${SRCREV} -DSUA_BUILD_NUMBER=${PR}"
+
+DEPENDS += "curl"
+DEPENDS += "dbus-glib"
+DEPENDS += "glib-networking"
+DEPENDS += "nlohmann-json"
+DEPENDS += "paho-mqtt-cpp"
+
+PACKAGES = "${PN}-data ${PN}"
+
+SYSTEMD_SERVICE:${PN} = "self-update-agent.service"
+
+FILES:${PN} += " \
+    ${systemd_system_unitdir}/self-update-agent.service \
+    ${bindir}/self-update-agent \
+    ${libdir}/libmini-yaml.so \
+"
+
+FILES:${PN}-data += " \
+    /data/self-update-images/ \
+"
+
+do_install() {
+    install -d ${D}${systemd_unitdir}/system/
+    install -m 0644 ${WORKDIR}/self-update-agent/self-update-agent.service ${D}${systemd_unitdir}/system
+
+    install -d ${D}${bindir}
+    install -m 755 ${B}/src/self-update-agent ${D}${bindir}
+
+    install -d ${D}${libdir}
+    install -m 755 ${B}/3rdparty/libmini-yaml.so ${D}${libdir}
```

Comment:
Are this files go without using FILES:${PN} +=

Reply:
$bindir and $libdir are in the default FILES:${PN} (see poky/meta/conf/bitbake.conf)

You need to list your files in FILES:$PN if you start overriding the default values, or if you start installing files into unknown or arbitrary locations. All the default locations are already handled by BitBake, such as /usr/bin